### PR TITLE
Crud service

### PR DESCRIPTION
**Pull request by dangph-1996**
==> Tittle: CRUD Servicre

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/15819

==> Attachment File:

### CODE REVIEW

**Comment by dangph-1996** [comment]
ready

File: app/controllers/services_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    @service = Service.new services_params
+    if @service.save
+      flash[:success] = t "success"
+      redirect_to @service
+    else
+      render :new
+    end
+  end
+
+  def show
```
Comment (phucmh-1356) [inline app/controllers/services_controller.rb]:
```suggestion
  def show; end
```

File: app/helpers/services_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module ServicesHelper
```
Comment (phucmh-1356) [inline app/helpers/services_helper.rb]: xóa file này đi nếu k dùng

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -7,4 +7,16 @@ def full_title page_title = ""
...
+  def get_index_page page = "page", key
+    params[page].to_i * Settings.end_stt + key + Settings.start_stt
+  end
+
+  def get_index key
+    key + Settings.start
+  end
+
+  def number_to_vnd number
+    number_to_currency(number,unit: "",separator: ",",delimiter: ".", precision: Settings.number_zero) + " " + t("unit")
```
Comment (phucmh-1356) [inline app/helpers/application_helper.rb]:
```suggestion
    number_to_currency(number, unit: "", separator: ",", delimiter: ".", precision: Settings.number_zero) + " " + t("unit")
```

**Comment by dangph-1996** [comment]
ready

**Pull request merged**